packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -271,42 +277,56 @@ export const menuTabs: ForumOptions<Array<MenuTab>> = {
-      tooltip: "Nate Soares writes about replacing guilt with other feelings and finding better ways to motivate yourself, so you can build a better future without falling apart.",
-      subItem: true,
-    }, {
-      id: 'most-important-century',
-      title: 'Most Important Century',
-      link: '/s/isENJuPdB3fhjWYHd',
-      tooltip: `Holden Karnofsky argues that we may be living in the most important century ever — a time when our decisions could shape the future for billions of years to come.`,
-      subItem: true,
-    }, {
-      id: 'takeAction',
-      title: 'Take Action',
-      link: `/${taggingNamePluralSetting.get()}/take-action`,
-      iconComponent: PlaylistAddCheck,
-      tooltip: "Opportunities to get involved with impactful work",
-      loggedOutOnly: true
+      customComponentName: "TabNavigationCollapsibleMenu",
+      customComponentProps: {
+        defaultExpanded: true,
+        items: [
+          {
+            id: 'handbook',
+            title: 'The EA Handbook',
+            link: '/handbook',
+            tooltip: "To help you learn the basics of Effective Altruism, we took some of the best writing and made this handbook. Think of it as the textbook you’d get in your first college course. It explains the core ideas of EA, so that you can start applying them to your own life.",
+            subItem: true,
+          }, {
+            id: 'replacing-guilt',
+            title: 'Replacing Guilt',
+            link: '/s/a2LBRPLhvwB83DSGq',
+            tooltip: "Nate Soares writes about replacing guilt with other feelings and finding better ways to motivate yourself, so you can build a better future without falling apart.",
+            subItem: true,
+          }, {
+            id: 'most-important-century',
+            title: 'Most Important Century',
+            link: '/s/isENJuPdB3fhjWYHd',
+            tooltip: `Holden Karnofsky argues that we may be living in the most important century ever — a time when our decisions could shape the future for billions of years to come.`,
+            subItem: true,
+          },
+        ],
+      },
     }, {
       id: 'events',
       title: 'Events',
       link: '/events',
       iconComponent: EventIcon,
       tooltip: 'Upcoming events near you',
       showOnMobileStandalone: true,
-      showOnCompressed: true
+      showOnCompressed: true,
+      customComponentName: "TabNavigationCollapsibleMenu",
+      customComponentProps: {
+        defaultExpanded: false,
+        items: [
+          {
+            id: "eventsList",
+            customComponentName: "EventsList",
+          },
+        ],
+      },
     }, {
-      id: 'eventsList',
-      customComponentName: "EventsList",
+      id: 'takeAction',
+      title: 'Take Action',
+      link: `/${taggingNamePluralSetting.get()}/take-action`,
+      iconComponent: PlaylistAddCheck,
+      tooltip: "Opportunities to get involved with impactful work",
+      loggedOutOnly: true
```

Comment:
I reordered this just to keep all the collapsible ones together

Reply:
Hmm I would put this higher up - at least above Events, but maybe even above Topics (if you want to keep the collapsible menu items together).

Also, would it be easy to add the sub-menu items back in for Community, now that we can just make it collapsed by default? (It was Local Groups, Online Groups, and Community Members, pointing to the three tabs on that page.) If not, then I can do that in a separate PR

Reply:
Yep, I can add those back in